test-framework/src/main/java/org/keycloak/test/framework/server/DistributionKeycloakTestServer.java

```diff
@@ -18,6 +18,7 @@ public class DistributionKeycloakTestServer implements KeycloakTestServer {
     @Override
     public void start(List<String> rawOptions) {
         keycloak = new RawKeycloakDistribution(DEBUG, MANUAL_STOP, ENABLE_TLS, RE_CREATE, REMOVE_BUILD_OPTIONS_AFTER_BUILD, REQUEST_PORT);
+        keycloak.copyProvider("org.keycloak.test", "providers");
```

Comment:
@vaceksimon this is nice solution, but I would expect some parameter, which will disable this option by default and also provide a variable for input parameters -> group, artifact, version.